docs/fields.asciidoc

```diff
@@ -638,6 +638,14 @@ type: keyword
 The result of the transaction. HTTP status code for HTTP-related transactions.
 
 
+[float]
+=== `transaction.sampled`
+
+type: boolean
+
+Unsampled transactions have no spans.
```

Comment:
Am I the only one who's brain explodes making sense of double negatives? _Sampled transactions have spans_.